dotnet/src/Connectors/Connectors.AzureOpenAI.UnitTests/Extensions/AzureOpenAIServiceCollectionExtensionsTests.cs

```diff
@@ -88,6 +89,25 @@ public void ServiceCollectionAddAzureOpenAITextEmbeddingGenerationAddsValidServi
 
     #endregion
 
+    #region Text to audio
+
+    [Fact]
+    public void ServiceCollectionAddAzureOpenAITextToAudioAddsValidService()
+    {
+        // Arrange
+        var sut = new ServiceCollection();
+
+        // Act
+        var service = sut.AddAzureOpenAITextToAudio("deployment-name", "https://endpoint", "api-key")
+            .BuildServiceProvider()
+            .GetRequiredService<ITextToAudioService>();
+
+        // Assert
+        Assert.IsType<AzureOpenAITextToAudioService>(service);
```

Comment:
Not needed.

```suggestion
```

Reply:
Discussed offline and agreed to keep it.